core/dataflow/src/main/java/edu/uci/ics/texera/dataflow/common/JsonSchemaHelper.java

```diff
@@ -51,8 +51,8 @@
     }
     
     public static void main(String[] args) throws Exception {
-        generateAllOperatorSchema();
-//        generateJsonSchema(ComparablePredicate.class);
+        //generateAllOperatorSchema();
+        generateJsonSchema(MysqlSourcePredicate.class);
```

Comment:
Don't change this file. Revert it to its original state after the schema.json is generated

Reply:
I have reverted it back to the original state